README.md

```diff
@@ -55,9 +61,18 @@ After the CloudFormation templates have been deployed, the [stack outputs](http:
 
 Stack is setup to launch stack in the us-west-2 (Oregon) region in your account:
 
-- from the root of your copy of the repo, run `aws s3 cp . s3://hacko-infrastructure-cfn --recursive --exclude ".git/*"`
-- copy the URL for the `master.yaml` file from S3
-- go to AWS CloudFormation - if creating new stack (e.g. for testing), choose "create stack"; if updating an existing stack, select that stack then click the *Update* button
+* from the root of your copy of the repo, run `aws s3 cp . s3://hacko-infrastructure-cfn --recursive --exclude ".git/*"`
+* copy the URL for the `master.yaml` (or other master-*.yaml template) file from S3
+* go to AWS CloudFormation - if creating new stack (e.g. for testing), choose "create stack"; if updating an existing stack, select that stack then click the *Update* button
+
+#### Security requirements
+
+The account of the AWS user who initially creates the stack requires many privileges in AWS, including:
+
+* IAM Role creation
+* IAM Policy creation
+
+Subsequent incremental Updates to an existing stack can sometimes be performed by AWS users with less privileges, depending on which stack objects are being created, updated or deleted.
```

Comment:
Probably worth mentioning that Cloud Formation will automatically rollback a stack change if the user who performed the updated didn't have adequate permissions. Just so people don't think they could possibly derail the state of the world.